src/js/tabs/osd.js

```diff
@@ -232,6 +232,23 @@ FONT.symbol = function (hexVal) {
 
 var OSD = OSD || {};
 
+OSD.getNumberOfProfiles = function() {
```

Comment:
I added the `osd_profile` to MSP that indicates the currently selected profile. If the value is 0 then that's an indication that `USE_OSD_PROFILES` is not defined and only the original single profile mode is available.

While I was there I also added the `osd_stick_overlay_radio_mode` so that can be added to the OSD tab at some point. Again, a value of 0 indicates the `USE_OSD_STICK_OVERLAY` feature is not available (actual value ranges from 1-4). This doesn't need to be implemented now, but if it is the return value for the selected OSD profile needs to be implemented before this.

https://github.com/betaflight/betaflight/pull/7503